concordium-node/src/configuration.rs

```diff
@@ -287,6 +287,13 @@ pub struct BakerConfig {
         env = "CONCORDIUM_NODE_BAKER_DECRYPT_CREDENTIALS"
     )]
     pub decrypt_baker_credentials: bool,
+    #[structopt(
+        long = "modules-cache-size",
+        help = "The maximum number of artifacts that can be stored in the module cache",
+        default_value = "100",
```

Comment:
Is this actually a good default?

Reply:
There are currently 28 on `mainnet` so it is certainly sufficient for the current state. However we should revise this in the future with more modules being deployed to the chain.